.gitlab-ci.yml

```diff
@@ -1218,6 +1216,8 @@ release_binary_tools:
       - $PUBLISH_ARTIFACTS == "true"
   stage: release
   image: debian:buster
+  tags:
+    - mender-qa-slave
```

Comment:
Is there any special reason for this? Since this is a very small job, isn't it just as well to run it in the public runners?

Reply:
Yes, let's open this for discussion. This got my attention last time that we run out of CI minutes, as mender-qa (integration) Pipeline also got stuck and I was not expecting that. So there is not really technical reason for doing this, but rather conceptually separating the individual pipelines from this one. Do you see any value on this?

As you can see it is not actually working (this job would, but the build mender-artifact is the one failing) and I have to modify the base image and experiment a bit more. Therefore I don't have an strong opinion and I can very well leave it as it was...

Reply:
Mostly cost saving/parallelism from my side. It's true that we can run out of minutes, but I still feel like this is a rarer event than all our mender-qa build hosts being busy. I think I would vote for leaving it as it was, but I'm fine with either.

Reply:
Make sense. But then I would move all the lightweight jobs to shared runners:
- init_workspace
- build_mender-cli
- build_mender-artifact (already dn shared runners)
- release_docker_images
- release_board_artifacts
- release_binary_tools

Any of these that you would be against?